test/e2e/service_latency.go

```diff
@@ -330,14 +331,14 @@ func singleServiceLatency(f *framework.Framework, name string, q *endpointQuerie
 	if err != nil {
 		return 0, err
 	}
-	framework.Logf("Created: %v", gotSvc.Name)
+	framework.Debugf("Created: %v", gotSvc.Name)
```

Comment:
there will be a few other PR's like this.  This one is the start to cleaning up extraneous logs, also verifying that the new logging utility works.